src/tests/test_buildercore_trop.py

```diff
@@ -229,6 +229,8 @@ def test_clustered_template(self):
             resources['EC2Instance1']['Properties']['Tags']
         )
         outputs = data['Outputs']
+        self.assertIn('ElasticLoadBalancer', list(outputs.keys()))
```

Comment:
does this work without the `list` wrapper? curious more than anything else

Reply:
I think it was a pattern to make the error message clearer by dumping a `list` rather than a `dict_keys` object.

Will follow-up to the comments with a separate PR this week.

Reply:
ah that makes sense